t/plugin/proxy-rewrite3.t

```diff
@@ -455,3 +455,99 @@ passed
 GET /echo HTTP/1.1
 --- response_headers
 test: test_in_set
+
+
+
+=== TEST 21: set route (test if X-Forwarded-Port can be set before proxy)
```

Comment:
This PR contains https://github.com/apache/apisix/pull/8404. We can deal with it after #8404 is merged.

Reply:
@spacewander could you check this PR again, the https://github.com/apache/apisix/pull/8404 is closed now, and all ci passed

Reply:
`X-Forwarded-For` is added like other `X-Forwarded-*` header